pythonforandroid/recipes/libxml2/__init__.py

```diff
@@ -0,0 +1,31 @@
+from pythonforandroid.toolchain import (CythonRecipe, shprint,
+                                        current_directory, info, Recipe)
+from pythonforandroid.patching import will_build, check_any
+import sh
+from os.path import join
+
+
+class Libxml2Recipe(CythonRecipe):
```

Comment:
This isn't a Cython module, so you'll want to just use `Recipe` as the base class.